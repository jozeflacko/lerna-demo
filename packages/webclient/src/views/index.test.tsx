import App from './index';
import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import * as ShallowRenderer from 'react-test-renderer/shallow';

describe("Testing Main App Component", ()=>{

    const renderer = ShallowRenderer.createRenderer();
    renderer.render(<App />);

    it("App will match snapshot",()=> {
        expect(renderer).toMatchSnapshot();
    });

    it('Parent is div',()=>{
        const result = renderer.getRenderOutput();
        expect(result.type).toBe('div');
    });

});
