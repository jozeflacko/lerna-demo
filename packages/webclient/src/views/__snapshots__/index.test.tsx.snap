// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Main App Component App will match snapshot 1`] = `
ReactShallowRenderer {
  "_context": Object {},
  "_element": <App />,
  "_forcedUpdate": false,
  "_instance": App {
    "context": Object {},
    "props": Object {},
    "refs": Object {},
    "state": null,
    "updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
  },
  "_newState": null,
  "_rendered": <div>
    Here is something:
    <br />
    <App
      prefixText="Current Time"
    />
  </div>,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": Array [],
    "_renderer": [Circular],
  },
}
`;
